src/extensions.web/Microsoft.Identity.Client.Extensions.Web/Microsoft.Identity.Client.Extensions.Web.csproj

```diff
@@ -0,0 +1,94 @@
+    <TargetFrameworkNetStandard>netstandard2.0</TargetFrameworkNetStandard>
+    <TargetFrameworkNetCore2>netcoreapp2.2</TargetFrameworkNetCore2>
+    <TargetFrameworkNetCore3>netcoreapp3.0</TargetFrameworkNetCore3>
+
+    <TargetFrameworks Condition="$([MSBuild]::IsOsPlatform('Windows'))">
+      $(TargetFrameworkNetDesktop);$(TargetFrameworkNetStandard);$(TargetFrameworkNetCore2)
+    </TargetFrameworks>
+    <TargetFrameworks Condition="$([MSBuild]::IsOsPlatform('Linux')) Or $([MSBuild]::IsOsPlatform('OSX')) Or '$(NetCoreOnly)' !='' ">
+      $(TargetFrameworkNetStandard);$(TargetFrameworkNetCore2)
+    </TargetFrameworks>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <!-- The MSAL.snk has both private and public keys -->
+    <DelaySign>false</DelaySign>
+  </PropertyGroup>
+
+  <PropertyGroup Label="NuGet and AssemblyInfo metadata">
+    <!--This should be passed from the VSTS build-->
+    <MsalClientSemVer Condition="'$(MsalClientSemVer)' == ''">4.0.0-localbuild</MsalClientSemVer>
+    <!-- For now, the Extensions.Web libraries are ALWAYS in preview mode -->
+    <MsalClientSemVer Condition="'$(MsalClientSemVer)' != ''">$(MsalClientSemVer)-preview</MsalClientSemVer>
+    <!--This will generate AssemblyVersion, AssemblyFileVersion and AssemblyInformationVersion-->
+    <Version>$(MsalClientSemVer)</Version>
+
+    <!-- Copyright needs to be in the form of © not (c) to be compliant -->
+    <Title>Microsoft Authentication Web Extensions Library for .NET</Title>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+    <Company>Microsoft Corporation</Company>
+    <Authors>Microsoft</Authors>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <PackageProjectUrl>https://go.microsoft.com/fwlink/?linkid=844761</PackageProjectUrl>
+    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+    <Description>
+      This package contains the binaries of the Microsoft Authentication Web Extensions Library for .NET (MSAL.NET).
+    </Description>
+    <PackageTags>Microsoft Authentication Library MSA MSAL B2C Azure Active Directory AAD Identity Authentication .NET Windows Store AspNet</PackageTags>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+    <Company>Microsoft Corporation</Company>
+    <Product>Microsoft Authentication Web Extensions Library</Product>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <EmbedUntrackedSources>true</EmbedUntrackedSources>
+    <!-- Build symbol package (.snupkg) to distribute the PDB containing Source Link -->
+    <IncludeSymbols>true</IncludeSymbols>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <DebugType>Full</DebugType>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(TargetFramework)' == '$(TargetFrameworkNetCore2)' ">
+    <DefineConstants>$(DefineConstants);NET_CORE_2</DefineConstants>
```

Comment:
Should we have consistent names for this constant between extension proj and msal proj? It might lead to confusion otherwise.

Reply:
I didn't want to initially mess with MSAL.  it will also need a netcore3 target (soon) so I agree that we should have a NET_CORE_2 target in MSAL.  Let's do that as a separate PR for changes to MSAL adding that target.  I definitely agree we want to avoid confusion.